contracts/upgrader/src/contract.rs

```diff
@@ -0,0 +1,99 @@
+use crate::error::ContractError;
+use soroban_sdk::{contract, contractimpl, Address, BytesN, Env, String, Symbol, Val};
+
```

Comment:
It would be great to convert the string to constants for maintainability and readability.

```
const MIGRATE: &str = "migrate";
const VERSION: &str = "version";
const UPGRADE: &str = "upgrade";

Symbol::new(&env, MIGRATE)
Symbol::new(&env, VERSION)
Symbol::new(&env, UPGRADE)
```